phpBB/install-config.yml

```diff
@@ -0,0 +1,35 @@
+installer:
```

Comment:
this file is a sample, part of the (to rite) documentation. Please move it to phpBB/docs/install-config.sample.yml